content/supermarket.md

```diff
@@ -74,6 +74,16 @@ Installing Private Supermarket
 To install a Private Supermarket, see the instructions
 [here](/install_supermarket/).
 
+Upgrading Private Supermarket
+------------------------------
+
+1. Get a backup of /var/opt/supermarket with the system shut down
+1. Read the release notes for any special considerations at github.com/chef/supermarket/RELEASE_NOTES.
+1. Get the new Supermarket package
+1. Upgrade your system with the new package
+1. `supermarket-ctl reconfigure`
+1. For good measure `systemctl list-units | grep runsvdir` then, using the unit name you find, do `systemctl restart UNIT_NAME`. This will restart the entire Supermarket software system from runsvdir, to the runsv and svlogd that are associated with each service process. This will avoid any mismatch between what is in memory and what is on disk for the Supermarket system.
```

Comment:
This should be split up. I may have the details wrong below, but this reads better as a set of instructions.
```suggestion
Private Supermarket should be updated on your server now. We recommend restarting the services that run Chef Supermarket to ensure that the old installation of Chef Supermarket doesn't persist in the server memory.

1. Get the name of the active unit: 

    `systemctl list-units | grep runsvdir` 

1. Restart the unit: 

    `systemctl restart UNIT_NAME`. 

This will restart the `runsvdir`, `runsv`, and `svlogd` service processes that run Chef Supermarket.
```